src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -178,6 +203,13 @@ private void UnloadDatabase(string databaseName)
             {
                 // this is already in the process of being deleted, we can just exit and let the other thread handle it
             }
+            catch (Exception e)
+            {
```

Comment:
We should rethrow here? Now we are ignoring all the exceptions.

Reply:
Yes, I don't like this. The error handling above is very explicit about why it is handling this. Why add this `catch`?

Reply:
done